### ISSUE TITLE
RC app crashes if Program & Manage is opened while WiFi is off.

### ISSUE THREAD

**Issue by FROGbots-4634** [FIRST-Tech-Challenge/SkyStone#9]
If you launch the RC app with WiFi off and open the Program & Manage screen, the app crashes. Here is the stacktrace:

```
2019-08-01 08:18:35.880 27630-27630/com.qualcomm.ftcrobotcontroller E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.qualcomm.ftcrobotcontroller, PID: 27630
    java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.qualcomm.robotcore.wifi.NetworkConnection.getPassphrase()' on a null object reference
        at org.firstinspires.ftc.robotserver.internal.webserver.CoreRobotWebServer.getConnectionInformation(CoreRobotWebServer.java:209)
        at org.firstinspires.ftc.robotcontroller.internal.FtcRobotControllerActivity.onOptionsItemSelected(FtcRobotControllerActivity.java:566)
        at org.firstinspires.ftc.robotcontroller.internal.FtcRobotControllerActivity$2$1.onMenuItemClick(FtcRobotControllerActivity.java:317)
        at android.widget.PopupMenu$1.onMenuItemSelected(PopupMenu.java:104)
        at com.android.internal.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:761)
        at com.android.internal.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:152)
        at com.android.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:904)
        at com.android.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:894)
        at com.android.internal.view.menu.MenuPopup.onItemClick(MenuPopup.java:128)
        at android.widget.AdapterView.performItemClick(AdapterView.java:310)
        at android.widget.AbsListView.performItemClick(AbsListView.java:1164)
        at android.widget.AbsListView$PerformClick.run(AbsListView.java:3139)
        at android.widget.AbsListView$3.run(AbsListView.java:4054)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6186)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
```

If, however, WiFi is turned off after the app has already been launched, then opening Program & Manage shows this:

![Screenshot_20190801-081814](https://user-images.githubusercontent.com/26417650/62292928-732b6180-b435-11e9-99e4-760b6aa2bcae.png)

**Comment by FROGbots-4634** [comment]
@ftctechnh why was this closed?

**Comment by AustinShalit** [comment]
It was resolved.

**Comment by NoahAndrews** [comment]
In the future, we should avoid putting the word "fixed" in front of a link to an issue on the public repo.

**Comment by ftctechnh** [comment]
@FROGbots-4634 
Hi FROGbots-4634 - Sorry about that... the public issue was linked to a PR on the private repo and when the pull request that prevents the app from crashing was merged into the master branch Git Hub automatically closed this issue.
The fix should be available in the soon-to-be-released v5.1 software.

Tom

**Comment by FROGbots-4634** [comment]
@ftctechnh ah, interesting, thanks for the clarification :)
